src/bin/nabu/main.rs

```diff
@@ -43,10 +43,13 @@ fn main() -> Result<()> {
 
     ctrlc::set_handler(move || {
         r.store(false, Ordering::SeqCst);
-    }).expect("Error setting Ctrl-C handler");
+    })
+    .expect("Error setting Ctrl-C handler");
 
     let level = if cli.debug { "debug" } else { "info" };
-    Logger::try_with_str(level)?.start()?;
+    let mut logger = Logger::try_with_str(level)?;
+    logger = logger.use_utc();
```

Comment:
Is there a way we can configure this during construction?

Reply:
Definitely, fixed in 07e0e0d26515feda14494512c11aa53b524a9902